tests/Guzzle7/phpstan.neon

```diff
@@ -6,5 +6,12 @@ parameters:
 	bootstrapFiles:
 		- vendor/autoload.php
 	reportUnmatchedIgnoredErrors: false
+	ignoreErrors:
+		-
+			message: '#^Method Prestashop\\ModuleLibGuzzleAdapter\\VersionDetection\:\:getGuzzleMajorVersionNumber\(\) has no return type specified.$#'
+			path: ../../src/VersionDetection.php
```

Comment:
If the other comment about the return type is taken in account, this ignored error could be removed

Reply:
Done